docker/Dockerfile.minimal

```diff
@@ -0,0 +1,13 @@
+FROM python:alpine
```

Comment:
Lets include this in the k8s folder with an explanation in the README, I'd like to keep maintain this dir as the place for GH image release